### ISSUE TITLE
Installing dockerfilelint-precommit-hooks fails on ARM64 (aarch)

### ISSUE THREAD

**Issue by f-skold** [pryorda/dockerfilelint-precommit-hooks#2]
I tried a configuration that worked on x86_64. On my arm64v8 board "Rock 4SE" it fails (se output at bottom).
I'm not sure on what project I should write this report, but I start with this one.

```
$ uname -a
Linux rockpi-4b 5.15.93-rockchip64 #23.02.2 SMP PREEMPT Fri Feb 17 23:48:36 UTC 2023 aarch64 aarch64 aarch64 GNU/Linux
```

 When disabling dockerfilelint-precommit-hooks the rest of the plugins installed correctly.

```
+    # - repo: https://github.com/pryorda/dockerfilelint-precommit-hooks
+    #   rev: v0.1.0
+    #   hooks:
+    #       - id: dockerfilelint
```

### version information

```
pre-commit version: 2.17.0
git --version: git version 2.34.1
sys.version:
    3.10.6 (main, May 29 2023, 11:10:38) [GCC 11.3.0]
sys.executable: /usr/bin/python3
os.name: posix
sys.platform: linux
```

### error information

```
An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/python3', '-mnodeenv', '--prebuilt', '--clean-src', '/home/username/.cache/pre-commit/repoyi3au1yz/node_env-default')
return code: 1
expected return code: 0
stdout: (none)
stderr:
     * Install node (20.4.0Traceback (most recent call last):
      File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
        return _run_code(code, main_globals, None,
      File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
        exec(code, run_globals)
      File "/usr/lib/python3/dist-packages/nodeenv.py", line 1042, in <module>
        main()
      File "/usr/lib/python3/dist-packages/nodeenv.py", line 890, in main
        create_environment(env_dir, opt)
      File "/usr/lib/python3/dist-packages/nodeenv.py", line 731, in create_environment
        install_node(env_dir, src_dir, opt)
      File "/usr/lib/python3/dist-packages/nodeenv.py", line 594, in install_node
        node_url = get_node_src_url(opt.node, get_node_src_url_postfix(opt))
      File "/usr/lib/python3/dist-packages/nodeenv.py", line 503, in get_node_src_url_postfix
        postfix_arch = arches[platform.machine()]
    KeyError: 'aarch64'

```

```
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pre_commit/error_handler.py", line 70, in error_handler
    yield
  File "/usr/lib/python3/dist-packages/pre_commit/main.py", line 396, in main
    return run(args.config, store, args)
  File "/usr/lib/python3/dist-packages/pre_commit/commands/run.py", line 416, in run
    install_hook_envs(to_install, store)
  File "/usr/lib/python3/dist-packages/pre_commit/repository.py", line 224, in install_hook_envs
    _hook_install(hook)
  File "/usr/lib/python3/dist-packages/pre_commit/repository.py", line 82, in _hook_install
    lang.install_environment(
  File "/usr/lib/python3/dist-packages/pre_commit/languages/node.py", line 97, in install_environment
    cmd_output_b(*cmd)
  File "/usr/lib/python3/dist-packages/pre_commit/util.py", line 154, in cmd_output_b
    raise CalledProcessError(returncode, cmd, retcode, stdout_b, stderr_b)
pre_commit.util.CalledProcessError: command: ('/usr/bin/python3', '-mnodeenv', '--prebuilt', '--clean-src', '/home/username/.cache/pre-commit/repoyi3au1yz/node_env-default')
return code: 1
expected return code: 0
stdout: (none)
stderr:
     * Install node (20.4.0Traceback (most recent call last):
      File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
        return _run_code(code, main_globals, None,
      File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
        exec(code, run_globals)
      File "/usr/lib/python3/dist-packages/nodeenv.py", line 1042, in <module>
        main()
      File "/usr/lib/python3/dist-packages/nodeenv.py", line 890, in main
        create_environment(env_dir, opt)
      File "/usr/lib/python3/dist-packages/nodeenv.py", line 731, in create_environment
        install_node(env_dir, src_dir, opt)
      File "/usr/lib/python3/dist-packages/nodeenv.py", line 594, in install_node
        node_url = get_node_src_url(opt.node, get_node_src_url_postfix(opt))
      File "/usr/lib/python3/dist-packages/nodeenv.py", line 503, in get_node_src_url_postfix
        postfix_arch = arches[platform.machine()]
    KeyError: 'aarch64'
```

**Comment by f-skold** [comment]
Started looking further at this. It seems like nodeenv could be the one where this is caused.
WIth ubuntu 22.04 (jammy), I seem to have a package nodeenv with version 0.13.4-1.1.

https://github.com/pre-commit/pre-commit/issues/732

(Same issue seem to be mentioned here: https://github.com/pre-commit/pre-commit/issues/732 )

Solved by this:

```
pip install -U pre-commit
pip install -U nodeenv
export PATH="$HOME/.local/bin:$PATH"
pre-commit run
```

**Comment by f-skold** [comment]
Solved with some manual steps
